.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can we use 4 spaces for formatting in the project? It's really difficult to see the indentations for me.